### ISSUE TITLE
Error after deleting a CustomFieldChoice

### ISSUE THREAD

**Issue by fabiana275** [digitalocean/netbox#576]
Hello,

I've used custom fields and some of them I have deleted.
But now when I create new tenant and trying to edit it after created I'm receiving that error:
" class 'extras.models.DoesNotExist'

CustomFieldChoice matching query does not exist. "
And go to home page

Could you please assist, how can I repair it?
netbox version: v1.6.1-r1

**Comment by fabiana275** [comment]
It also happening for the devices which are added recently!

**Comment by jeremystretch** [comment]
Could you please describe exactly what actions you took to trigger the suspected bug so that we can attempt to replicate?

**Comment by jeremystretch** [comment]
It sounds like you might have created a custom field with choices, and then deleted all the choices for that field. Is that correct? If so, you should be able to resolve this by creating at least one choice for the field, or by simply deleting the custom field.

**Comment by fabiana275** [comment]
Hi,

There was 2 or more selection choices and i have delete some of them. Now in the custom fields there is at least one choice per field. I have tried to re-create the choices which I have deleted but without success.
After your answer i tried to delete the selection fields, but I have received the following error:

"class 'django.utils.encoding.DjangoUnicodeDecodeError'

'ascii' codec can't decode byte 0xd0 in position 12: ordinal not in range(128). You passed in <CustomFieldValue: [Bad Unicode data]> (<class 'extras.models.CustomFieldValue'>)   "
